sound/sound_library.h

```diff
@@ -8,6 +8,7 @@ enum SayType {
   SAY_WHOLE,
 };
 
+#ifdef SAY_COLOR_LIST
```

Comment:
if sound_library is used and we don't use SAY_COLOR_LIST save memory for EFFECT and enum